test/e2e/common_test.go

```diff
@@ -103,6 +103,7 @@ func TestLibpod(t *testing.T) {
 
 var _ = SynchronizedBeforeSuite(func() []byte {
 	// make cache dir
+	ImageCacheDir = filepath.Join(os.TempDir(), "imagecachedir")
 	if err := os.MkdirAll(ImageCacheDir, 0777); err != nil {
```

Comment:
The permissions on this directory seems to be too loose.  I don't think we should allow different users on the system to share the ImageCacheDir, since it could be an attack route to attack each other.
This should be per user.

Reply:
Change ImageCacheDir permissions from `0777` to `0700` .
As far as I can see, I think this change is no ploblem.